### DISCUSSION TITLE
Yellow taxi can do inversion

### DISCUSSION THREAD

**Discussion by Coasterhacker** [OpenRCT2/OpenRCT2#23374] [Feature requests]
![Test Coaster and CTR 2024-12-10 13-42-40](https://github.com/user-attachments/assets/3ceae9ea-3b27-47aa-ae92-356bfc0cc27f)
what's weird about it. only yellow taxi can do inversion. is this possible without a cheat on?

**Comment by SpartanFrederic104** [comment]
These trains have the same number of sprites and sprite groups as the Corkscrew Roller Coaster's trains, which indicates the Yellow Taxi Trains may have originally been designed for the Corkscrew Roller Coaster but were instead put on the Giga Coaster.

Additionally, if you have the Yellow Taxi Trains available, they will enable vertical slopes on the Giga Coaster.
